modules/cached/lib/impl.js

```diff
@@ -84,6 +84,24 @@ module.exports = function (config, models) {
         return { rows: orgids, count }
     };
 
+    const saveBlockNumber = (value = 0) => models.stats.upsert({
+        name: 'blockNumber',
+        value
+    });
+
+    const getBlockNumber = async () => {
+        const record = await models.stats.findOne({
+            where: {
+                name: {
+                    [Op.eq]: 'blockNumber'
+                }
+            }
+        });
+        const value = record && record.value ? Number(record.value) : 0;
+        return String(value) !== 'NaN' ? value : 0;
```

Comment:
No, this issue does not give us any benefits )
Because it does not matter ever we subscribing from block `0` or from specific contract creation block in both cases subscription will return us ALL events from the smart contract.
The number I have mentioned in the PR description - is just random block number from these that had been mined last night so this way we can be sure that all events generated before this block will not be obtained.

Reply:
Indeed :)